README.md

```diff
@@ -238,21 +252,25 @@ In the context of broadcasting, only the following model events can be broadcast
 - `trashed` _if soft delete is enabled_
 - `restored` _if soft delete is enabled_
 
-By default, all of these events are broadcasted, but you can define which events in particular you'd like to broadcast using the `broadcastEvents` method on the model itself:
+By default, all of these events would broadcast, but you can define which events in particular you'd like to broadcast using the `broadcastOn` method on the model itself, just like Laravel suggest it:
 
 ```php
 /**
- * Define the model events to broadcast.
+ * Get the channels that model events should broadcast on.
  *
- * @return array
+ * @param  string  $event
+ * @return \Illuminate\Broadcasting\Channel|array
  */
-public function broadcastEvents()
+public function broadcastOn($event)
 {
-    return ['created', 'updated'];
+    return match ($event) {
+        'deleted' => [], // disable broadcasting of the 'deleted' model event
+        default => ['users'],
+    };
 }
 ```
 
-Now only the `created` and `updated` events for this Model will broadcast.
+Now only the `created` and `updated` events for this Model will broadcast (if soft delete is disabled).
```

Comment:
I'm not sure I understand this 🤔  How can it only broadcast created and updated events if you only disable deleted?

I think with the update this doesn't make sense as looking at the old code in the diff it does.

Also on this - the structure of the match() function being an exclude rather than an include is a bit confusing to me but might just be me.

Reply:
This is referring to the [official documentation](https://laravel.com/docs/8.x/broadcasting#model-broadcasting)

![Image 2021-07-28 at 4 53 30 pm](https://user-images.githubusercontent.com/668419/127344578-290c5549-e7a8-4e75-ab9d-f0be6a0b3013.jpg)

> How can it only broadcast created and updated events if you only disable deleted?
Because it would fallback to use whatever you have defined in your `default`.
This would be similar to have
```php
switch($event) {
    case 'deleted':
        return [];
    default:
        return ['users'];
}
```
Which means for `deleted` we don't define any channel but for any other event we do define some.

Reply:
I think we need to adjust this text then:
`Now only the 'created' and 'updated' events for this Model will broadcast (if soft delete is disabled).`

To something like `Now the soft delete event will not be broadcast`

Reply:
Not very elegant English 😅  but the old text implied that more than just deleted was excluded.

Reply:
To be honest this bit of text is now gone and I just point to the official documentation.